packages/directory/addon/breakpoints.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
What is this for? Is this to make the table more responsive to screen size?

Reply:
Yes, it's a file used by ember-responsive which is included by ember-light-table